spec/services/importers/create_cohort_spec.rb

```diff
@@ -6,21 +6,20 @@
   let(:csv) { Tempfile.new("data.csv") }
   let(:path_to_csv) { csv.path }
 
+  let(:start_year) { Cohort.ordered_by_start_year.last.start_year + 50 }
+
   subject(:importer) { described_class.new(path_to_csv:) }
 
   describe "#call" do
     context "with new cohorts" do
       before do
         csv.write "start-year,registration-start-date,academic-year-start-date,npq-registration-start-date"
         csv.write "\n"
-        csv.write "2020,2020/05/10,2020/09/01,"
-        csv.write "\n"
-        csv.write "2021,2021/05/10,2021/09/01,"
-        csv.write "\n"
-        csv.write "2022,2022/05/10,2022/09/01,"
-        csv.write "\n"
-        csv.write "2023,2023/05/10,2023/09/01,"
-        csv.write "\n"
+        4.times.each do |n|
```

Comment:
suggestion: I prefer the explicitness of the old test, where exact cohorts are specified and tested against. Having relative cohorts makes it harder to understand the test and debug.

Reply:
update I guess we can't work the explicit ones in the new world

Reply:
Moved to explicitly using 3031-4 and 4041 as the years in the tests